data/human/human missions.txt

```diff
@@ -6301,3 +6301,418 @@ mission "Blind Man from Martini"
+			`	You don't even think about the pay until you're back on board, when you see an incoming message that informs you <payment> has been transferred to your account, along with a heartfelt thank-you note from both men.`
+
+event "tom and carl's wedding"
+
+event "carl elected governor"
+
+event "tom and carl's child"
+
+mission "Take Me To The South D: Followup 1"
+	landing
+	source
+		near "Kappa Centauri" 1 100
+	to offer
+		has "event: tom and carl's wedding"
+	on offer
+		conversation
+			`You receive a message, and when you open it, you see a beautifully produced set of photos of Tom Ardwright's wedding to Carl Chamberlain on Cornucopia. It includes a letter from the two of them:`
+			`	"Dear <first> <last>,`
+			`	"We are delighted to announce that despite the many hardships around us in these times, Tom Ardwright and Carl Chamberlain were married three days ago at the Hart's Hill Vineyard on Cornucopia. This would not have been possible without your willingness to help a man in need, so we thank you from the bottom of our hearts, and if you are ever on Cornucopia, we would be happy to host you. Please enjoy these photos from the wedding.`
+			`	"Tom & Carl Ardwright-Chamberlain"`
+			`	You can easily see Tom's hand in the design of the photo set, which includes a photo of them on the tarmac at the Cornucopia spaceport with a note that that was the spot where they were reunited after the uncertainty of the war parted them.`
+				decline
+
+mission "Take Me To The South D: Followup 1A"
+	landing
+	source "Cornucopia"
+	to offer
+		has "Take Me To The South D: Followup 1: declined"
+		not "event: carl elected governor"
+	on offer
+		"visited tom and carl" ++
+		conversation
+			`When you're making your approach for landing, doing your normal back and forth with the controller at the spaceport, you're surprised to hear them ask you to divert to a special pad. You do so, and when you land, you find Tom, the man you brought south from the Core, and his husband Carl, the Agricultural Minister of Cornucopia, waiting for you.`
+			`	They greet you like an old friend (though you didn't speak two words to Carl when you were here), and insist on serving you dinner at their house. For a prominent government minister's house, it's remarkably modest. The meal, on the other hand, is anything but: it's probably the best food you've had, which shouldn't be a surprise from a planet that's almost entirely covered in farms. You swap stories for a while - you telling about your "thrilling adventures in space", as Tom describes them, and them telling about some of their much more ordinary, but no less interesting, times here on Cornucopia. They also hint about some big things coming, sharing a knowing look but saying no more.`
```

Comment:
Hm. Can you be more specific? I can see that in describing the house as "modest" without giving any indication as to why/how, but the describing a meal as being very good doesn't strike me as telling not showing, and to go into more detail about the stories being told seems...unnecessarily verbose for a vignette like this?

Reply:
A quick draft at rewriting this paragraph.
```suggestion
			`	They greet you like an old friend (despite never having talked with Carl before), and insist on serving you dinner. Rather than the grand mansions of other politicians, their house is a simple bungalow that wouldn't look out of place in a suburb. The meal, on the other hand, is anything but modest: there are plates of kebabs, pastas, stir fries, and cakes, made from ingredients sourced from all over the planet. You swap stories for a while - you with your "thrilling adventures in space", as Tom describes them, and them with their much more ordinary, but no less interesting, times here on Cornucopia. They also hint at some big things coming, but reveal nothing more than a knowing look.`
```

Reply:
OK, that does help. Thanks; I'll incorporate this.